sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/pom.xml

```diff
@@ -26,4 +26,95 @@
       <version>2.6.0-beta.1</version><!-- {x-version-update;com.azure.spring:azure-spring-cloud-feature-management-web;current} -->
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <!-- START: Empty Java Doc and Sources -->
+      <!-- The following code will generate an empty javadoc with just a README.md. This is necessary
+            to pass the required checks on Maven. The way this works is by setting the classesDirectory
+            to a directory that only contains the README.md, which we need to copy. If the classesDirectory
+            is set to the root, where the README.md lives, it still won't have javadocs but the jar file
+            will contain a bunch of files that shouldn't be there. The faux sources directory is deleted
+            and recreated with the README.md being copied every time to guarantee that, when building locally,
+            it'll have the latest copy of the README.md file.
+      -->
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>3.3.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-javadoc-plugin;external_dependency} -->
+          <executions>
+              <execution>
+                  <id>attach-javadocs</id>
+                  <goals>
+                      <goal>jar</goal>
+                  </goals>
+                  <configuration>
+                      <skip>true</skip>
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-antrun-plugin</artifactId>
+          <version>1.8</version> <!-- {x-version-update;org.apache.maven.plugins:maven-antrun-plugin;external_dependency} -->
+          <executions>
+          <execution>
+            <id>empty-javadoc-jar-with-readme</id>
+            <phase>package</phase>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <classifier>javadoc</classifier>
+              <classesDirectory>${project.basedir}/javadocTemp</classesDirectory>
+            </configuration>
+          </execution>
+          <execution>
+            <id>empty-source-jar-with-readme</id>
+            <phase>package</phase>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <classifier>sources</classifier>
+              <classesDirectory>${project.basedir}/sourceTemp</classesDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
I think this was supposed to be executions for maven-jar-plugin?

Reply:
Yeah, I've pushed a change to fix this. Yay copy and paste.